src/routes/login/+page.svelte

```diff
@@ -29,48 +122,60 @@
                 <span>Email</span>
             </label>
             <input
-                class="input {emailIndicator}"
+                class="bg-surface-700 text-white border rounded-full transition-colors {emailIndicator} w-full focus:border-primary-500 focus:outline-none focus:ring-0"
                 type="text"
                 id="email_input"
                 placeholder="beispiel.organisation@mail.com"
+                bind:this={emailReference}
             />
+            {#if emailIndicator == indicatorStatus.warning}
+                <div
+                    class="flex flex-col justify-center items-center transition-opacity"
+                >
+                    <p class="text-sm text-warning-500">
+                        Bitte gib eine valide Email-Adresse ein
+                    </p>
+                </div>
+            {/if}
             <!-- Password field -->
             <label class="label" for="password_input">
                 <span>Passwort</span>
             </label>
-            <div
-                class="input-group input-group-divider relative grid-cols-[auto_1fr_auto]"
-            >
+            <div class="relative">
                 <input
-                    class="w-72 {passwordIndicator}"
+                    class="bg-surface-700 text-white border rounded-full transition-colors {passwordIndicator} w-full focus:border-primary-500 focus:outline-none focus:ring-0"
```

Comment:
Siehe Anmerkung bei Email input.
```suggestion
                    class="input {passwordIndicator}"
```